src/pybind/mgr/dashboard/frontend/src/styles/defaults/_bootstrap-defaults.scss

```diff
@@ -33,6 +33,14 @@ $danger: $red !default;
 $light: $gray-100 !default;
 $dark: #777 !default;
 
+//badges colors
+$green-300: #6ec664;
+$cyan-300: #009596;
+$purple-300: #a18fff;
+$light-blue-300: #35caed;
+$gold-300: #f4c145;
+$light-green-300: #ace12e;
+
```

Comment:
If we only use these for badges we could customize with names like label-color-mds. Could also help when mapping the color keys